.clippy.toml

```diff
@@ -0,0 +1 @@
+cognitive-complexity-threshold = 252
```

Comment:
instead, please ignore in the code directly

Reply:
#[allow(clippy::cognitive_complexity)]

Reply:
Fixed